maui-toolkit/Bottom-Sheet/BottomSheet-Customization.md

```diff
@@ -67,11 +67,11 @@ Content = bottomSheet;
 {% endtabs %}
 
 ## Allowed State
-The states of the `BottomSheet` can be controlled using the `AllowedState` property. The default value is `All`, allowing all states. This property accepts the following values:
+The states of the `BottomSheet` allow you to control the height of the bottom sheet using the `AllowedState` property. The default value is `All`, which allows transitions between all available states. This property accepts the following values:
```

Comment:
The BottomSheet allows to control the transition between the states using the `AllowedState`  property. The default value is `All`, which allows transitions between all available states. This property accepts the following values: